### PR TITLE
This is our file abdulbasit And shadan

### PR DESCRIPTION
**Pull request by abdulbasit-dev**
me and shadan do exercice
here is contact for shadan email or github username
Email;
Shadanabdulkarim94@gmail.com
Github username:
Shadan72

### CODE REVIEW

**Comment by abdulbasit-dev** [comment]
complete the block post
with shadan

**Comment by abdulbasit-dev** [comment]
complete the task abdulabsit-shadan

**Comment by abdulbasit-dev** [comment]
complete the Blog post website (Abdulbasit)

File: shadan-abdulbasit/style.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+body{
+  margin:0px;
+  padding:0px;
+  font-family:"Helvetica";
+}
+.container{
+  display: flex;
+
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:8]: try not leave new lines at the end of your css block

File: shadan-abdulbasit/style.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+aside div {
+  padding-left: 30px;
+  position:fixed; 
+
+  
+}
+
+aside div p a{
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:32]: try to use classes instead of tag selectors

File: shadan-abdulbasit/style.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  flex:7;
+  padding: 2.5em;
+}
+#blog{
+  margin-bottom:-12px;
+  font-size:28px;
+  /* font-size: 24px; */
+}
+
+#pined-post , #resent-post{
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:52]: you can here create a class with name for example *margin-bottom* and then give these to elements this class

File: shadan-abdulbasit/style.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  padding: 3px 17px;
+  border-radius: 4px;
+  margin: 0 3px;
+  
+}
+
+#css{background: #5DB85B;}
+#bg-blue{background: #4C85D5;}
+#bg-red{background: #E5284B;}
+#not-visted{background: #9e9e9e;}
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:103]: change those IDs into classes 
```
#css{background: #5DB85B;}
#bg-blue{background: #4C85D5;}
#bg-red{background: #E5284B;}
#not-visted{background: #9e9e9e;}

.css{background: #5DB85B;}
.bg-blue{background: #4C85D5;}
.bg-red{background: #E5284B;}
.not-visted{background: #9e9e9e;}
```
and then fic the html to change the *ID* property into *Class* property

File: shadan-abdulbasit/style.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  margin-right: 20px;
+  border: 1px solid gray;
+  border-radius: 2px;
+ 
+}
+
+
+.img1{
+  flex: 1;
+  /* border:2px solid cadetblue; */
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:125]: don't leave random comments in your PR, so remove this one

File: shadan-abdulbasit/style.css [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  
+ 
+  
+}
+.img2{
```
Comment (MohammedAlabd) [inline shadan-abdulbasit/style.css:130]: the classes `.img1` and `.img2` has exactly the same content which is `flex: 1;` so you can remove one of those classes and use the other class for both images

**Review by MohammedAlabd — CHANGES_REQUESTED** [review]
you are doing amazing job 🤩 keep going,

I have only few comments below please fix them and I will merge your PR

File: shadan-abdulbasit/style.css (hunk 2 above)
Comment (abdulbasit-dev) [reply shadan-abdulbasit/style.css:32]: why? isn't the some?

**Pull request closed without merging**